hydra-cluster/exe/log-filter/Main.hs

```diff
@@ -36,24 +36,34 @@ logFilterOptions =
     (logFilterOptionsParser <**> helper)
     ( fullDesc
         <> progDesc
-          "Filter and trim down logs from a Hydra node. This filter \
-          \ keeps only the 'Node' messages, removes the networking layer ones, \
-          \ and replaces full transactions in the log entries by transaction ids."
-        <> header "log-filter - Raw Hydra-node logs filter"
+          ( toString $
+              unlines
+                [ "Filter logs and compute events duration per transaction."
```

Comment:
Is it still doing any filtering?

Reply:
It is actually a `map` + `filter` so I would say yes, but I am fine renaming it. What would be a good name?